### ISSUE TITLE
Taiga opens videos in default file player.(Doesn't recognize MPC?)

### ISSUE THREAD

**Issue by Thomasedv** [erengy/taiga#134]
So i have VLC and MPC-HC (with Smooth Video Project) but prefer having VLC as default for mkv files. Only if i set MPC as the default media player will it open those types, a mp4 video still opens in VLC too. VLC is not selected in the settings, MPC is. Problem seems to be that MPC-HC doesn't get recognized since the small icon does not get color like VLC and windows media player in the settings. I also checked the other MPC options and tried running as moderator. Running windows 8.1 and got latest updates. 

Edit: I think the problem is there because i have it saved in my SVP folder on another harddrive which the isn't on. As usual C: got OS, D: is SVP/MPC. Taiga is also on disk D: Still not sure how to solve that though if that is the problem.

**Comment by erengy** [comment]
Taiga indeed opens videos in the default application associated with video files. There's no setting to change this. The relevant page (*Settings → Recognition → Media players*) just allows you to enable/disable detection of certain players, nothing more.

MPC-HC's icon not being colored is not an issue. It shows up gray because yours is installed under an uncommon directory. This doesn't affect anime recognition in any way; it's just cosmetics.

I just tried installing Smooth Video Player and it worked as expected. When I opened a video in the player (MPC-HC) that comes along with SVP, Taiga detected the player and identified the anime without any problem.
